README.md

```diff
@@ -138,7 +138,12 @@ RspecApiDocumentation.configure do |config|
 
   # Change how the post body is formatted by default, you can still override by `raw_post`
   # Can be :json, :xml, or a proc that will be passed the params
-  config.post_body_formatter = Proc.new { |params| params }
+  config.response_body_formatter = Proc.new { |params| params }
```

Comment:
@jakehow & @oestrich sorry - I edited the original commit - which might not be too obvious via github notifications. I fixed the mistake in documentation. Is this ok to merge now?